gradle.properties

```diff
@@ -1,4 +1,4 @@
-projectVersion=3.3.0-SNAPSHOT
+projectVersion=3.4.0-SNAPSHOT
 projectGroup=io.micronaut.cache
 micronautDocsVersion=2.0.0
 micronautVersion=3.3.4
```

Comment:
update micronaut to 3.4.3 and point githubCoreBranch to 3.5.0

Reply:
https://github.com/micronaut-projects/micronaut-cache/pull/459/commits/0916977e3e7cb88cb0bb4242547a332beaa735da